### PR TITLE
Refactor avahi and bluez instructions

### LINKED ISSUE #24: Add commands to install avahi-daemon and bluez
Installation commands are missing for Ubuntu Server / Desktop.

---
*Reported from: https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/otbr-on-ubuntu/*

### LINKED ISSUE #26: Remove bluez and avahi-observe connection from Chip Tool instructions
These are now auto connected.

Related to https://github.com/canonical/chip-tool-snap/issues/56

Additionally affected pages:
- https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/matter-and-thread-on-ubuntu/
- https://canonical-matter.readthedocs-hosted.com/en/latest/tutorial/pi-gpio-commander/

---
Reported from: https://canonical-matter.readthedocs-hosted.com/en/latest/how-to/chip-tool-commission-and-control/*

### PR DESCRIPTION
**Pull request by farshidtz**
Related issues:
- Close #24
- Close #26

### CODE REVIEW

**Comment by farshidtz** [comment]
@jpm-canonical to serve locally:
```
make run
```
and hope that there are no dependency issues.

File: how-to/chip-tool-commission-and-control.rst [hunk 1]
```diff
@@ -29,39 +47,19 @@ Install the Chip Tool snap:
...
-
-   On **Ubuntu Core**, the ``avahi-observe`` and ``bluez`` interfaces 
-   are not provided by the system.
+Once installed, the application should be available as ``chip-tool`` on your machine.
 
-   These interfaces are provided by other snaps, such as the
-   `Avahi <https://snapcraft.io/avahi>`_ and
-   `BlueZ <https://snapcraft.io/bluez>`_ snaps. To install the snaps
-   and connect to the interfaces, run:
+The snap restricts the app's access to necessary resources on the host.
```
Comment (jpm-canonical) [inline how-to/chip-tool-commission-and-control.rst:52]:
```suggestion
The snap restricts the app's access to only the necessary resources on the host.
```

File: how-to/chip-tool-commission-and-control.rst [hunk 2]
```diff
@@ -29,39 +47,19 @@ Install the Chip Tool snap:
...
+The access to host is managed via `snap interface <https://snapcraft.io/docs/interface-management>`_ connections.
```
Comment (jpm-canonical) [inline how-to/chip-tool-commission-and-control.rst:53]:
```suggestion
This access can be managed via `snap interface <https://snapcraft.io/docs/interface-management>`_ connections.
```

File: how-to/chip-tool-commission-and-control.rst [hunk 3]
```diff
@@ -29,39 +47,19 @@ Install the Chip Tool snap:
...
 
-   .. code:: shell
+By default, the snap auto connects the following interfaces:
```
Comment (jpm-canonical) [inline how-to/chip-tool-commission-and-control.rst:55]:
```suggestion
By default the snap auto connects the following interfaces:
```

**Review by jpm-canonical — CHANGES_REQUESTED** [review]

**Comment by jpm-canonical** [comment]
A general remark I have about this set of how-to's on matter is that there is a lot of duplication. Especially when it comes to chip-tool. It's definitely not something to address now, but something to think about.

File: how-to/chip-tool-commission-and-control.rst (hunk 3 above)
Comment (farshidtz) [reply how-to/chip-tool-commission-and-control.rst:55]: "By default" is an introductory phrase, which is usually followed by a comma to improve readability. I did some research and this appears to be common practice.

**Review by jpm-canonical — APPROVED** [review]

**Pull request merged**